exchanges/bybit/bybit_websocket.go

```diff
@@ -54,6 +55,24 @@ const (
 	websocketPrivate = "wss://stream.bybit.com/v5/private"
 )
 
+var defaultSubscriptions = subscription.List{
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.TickerChannel},
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.OrderbookChannel, Levels: 50},
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.AllTradesChannel},
+	{Enabled: true, Asset: asset.Spot, Authenticated: true, Channel: subscription.MyOrdersChannel},
```

Comment:
```suggestion
	{Enabled: true, Asset: asset.Spot, Channel: subscription.CandlesChannel, Interval: kline.OneHour},
	{Enabled: true, Asset: asset.Spot, Authenticated: true, Channel: subscription.MyOrdersChannel},
```

Reply:
Happy to add this.
You realise it wasn't enabled before, right?

I'll update the PR notes.

Fixed 0acd68ea6